cea/demand/occupancy_model.py

```diff
@@ -84,8 +85,8 @@ def calc_schedules(list_uses, archetype_schedules, bpr, archetype_values, stocha
         for schedule in ['people', 've', 'Qs', 'X', 'Vww', 'Vw']:
             schedules[schedule] = np.zeros(HOURS_IN_YEAR)
         # electricity and process schedules may be greater than 0
-        for schedule in ['Ea', 'El', 'Qcre', 'Ed', 'Epro', 'Qhpro']:
-            codes = {'Ea': 1, 'El': 1, 'Ed': 1, 'Epro': 3, 'Qhpro': 3,'Qcre': 3}
+        for schedule in ['Ea', 'El', 'Qcre', 'Ed', 'Epro', 'Qhpro', 'Qcpro']:
+            codes = {'Ea': 1, 'El': 1, 'Ed': 1, 'Qcre': 1, 'Epro': 3, 'Qhpro': 3, 'Qcpro': 3}
```

Comment:
Why Qcre changed?, I believe it is 3. It should not be accounted as an internal load
COOLROOMS
SERVERS
QCPRO
QHPRO
are all isolated from the R-C model under the assumption that they have their own HVAC system. Maybe I am missing something?

Reply:
Cool rooms don't have process schedules in CEA. In fact, refrigeration was already calculated based on the electricity schedule before I mistakenly changed it a while back.